python/paddle/nn/functional/vision.py

```diff
@@ -12,80 +12,113 @@
-from ...fluid.layers import psroi_pool  #DEFINE_ALIAS
-from ...fluid.layers import resize_bilinear  #DEFINE_ALIAS
-from ...fluid.layers import resize_nearest  #DEFINE_ALIAS
-from ...fluid.layers import resize_trilinear  #DEFINE_ALIAS
-from ...fluid.layers import roi_align  #DEFINE_ALIAS
-from ...fluid.layers import roi_pool  #DEFINE_ALIAS
-from ...fluid.layers import space_to_depth  #DEFINE_ALIAS
-from ...fluid.layers import yolo_box  #DEFINE_ALIAS
-from ...fluid.layers import yolov3_loss  #DEFINE_ALIAS
-
-from ...fluid.layers import fsp_matrix  #DEFINE_ALIAS
-from ...fluid.layers import image_resize_short  #DEFINE_ALIAS
-from ...fluid.layers import pixel_shuffle  #DEFINE_ALIAS
-from ...fluid.layers import retinanet_detection_output  #DEFINE_ALIAS
-from ...fluid.layers import retinanet_target_assign  #DEFINE_ALIAS
-from ...fluid.layers import roi_perspective_transform  #DEFINE_ALIAS
-from ...fluid.layers import shuffle_channel  #DEFINE_ALIAS
+from ...fluid.data_feeder import check_variable_and_dtype
+from ...fluid.layer_helper import LayerHelper
+from ...fluid.framework import in_dygraph_mode
 
 __all__ = [
-    'affine_channel',
-    'affine_grid',
-    'anchor_generator',
-    'bipartite_match',
-    'box_clip',
-    'box_coder',
-    'box_decoder_and_assign',
-    'collect_fpn_proposals',
+    #'affine_channel',
+    #'affine_grid',
+    #'anchor_generator',
+    #'bipartite_match',
+    #'box_clip',
+    #'box_coder',
+    #'box_decoder_and_assign',
+    #'collect_fpn_proposals',
     #       'deformable_conv',
-    'deformable_roi_pooling',
-    'density_prior_box',
-    'detection_output',
-    'distribute_fpn_proposals',
-    'fsp_matrix',
-    'generate_mask_labels',
-    'generate_proposal_labels',
-    'generate_proposals',
-    'grid_sampler',
-    'image_resize',
-    'image_resize_short',
+    #'deformable_roi_pooling',
+    #'density_prior_box',
+    #'detection_output',
+    #'distribute_fpn_proposals',
+    #'fsp_matrix',
+    #'generate_mask_labels',
+    #'generate_proposal_labels',
+    #'generate_proposals',
+    #'grid_sampler',
+    #'image_resize',
+    #'image_resize_short',
     #       'multi_box_head',
     'pixel_shuffle',
-    'prior_box',
-    'prroi_pool',
-    'psroi_pool',
-    'resize_bilinear',
-    'resize_nearest',
-    'resize_trilinear',
-    'retinanet_detection_output',
-    'retinanet_target_assign',
-    'roi_align',
-    'roi_perspective_transform',
-    'roi_pool',
-    'shuffle_channel',
-    'space_to_depth',
-    'yolo_box',
-    'yolov3_loss'
+    #'prior_box',
+    #'prroi_pool',
+    #'psroi_pool',
+    #'resize_bilinear',
+    #'resize_nearest',
+    #'resize_trilinear',
+    #'retinanet_detection_output',
+    #'retinanet_target_assign',
+    #'roi_align',
+    #'roi_perspective_transform',
+    #'roi_pool',
+    #'shuffle_channel',
+    #'space_to_depth',
+    #'yolo_box',
+    #'yolov3_loss'
 ]
+
+
+def pixel_shuffle(x, upscale_factor, data_format="NCHW", name=None):
+    """
```

Comment:
no need for duplicated docstrings for functional version, just refer to class doc

Reply:
Yep, It is the same thing.

Reply:
literally, just remove the duplicated description (bug keep the args and stuff) and  add a reference to the Layer version.